src/humancellatlas/data/metadata/helpers/staging_area.py

```diff
@@ -219,20 +215,23 @@ def _entity_ids_by_type(self,
 
 @attr.s(frozen=True, kw_only=True, auto_attribs=True)
 class CannedStagingAreaFactory:
-    #: The local path to the repository clone
+    #: The local path to the repository
     local_path: Path
 
     @classmethod
     def clone_remote(cls, remote_url: furl, local_path: Path, ref: str) -> Self:
         """
-        :param remote_url: A GitHub repository clone URL (HTTPS) a
-                 git ref, and the relative path inside the repository to the
-                 base of the staging area.
+        Clone a Git repository into a local path.
+
+        :param remote_url: A GitHub repository clone URL (HTTPS)
```

Comment:
Not necessarily a GitHub repository, either. Do you not recall that we we would make this class unaware of the GitHub specifics? Please take a step back and review all docstrings with respect to the separation of concerns as we discussed during one of our PL slots.